### DISCUSSION TITLE
[javascript deep dive] 13장. 스코프

### DISCUSSION THREAD

**Discussion by rladpwl0512** [woowacourse-study/2022-javascript-bank#19] [General]
[javascript deep dive] 13장. 스코프

**Comment by jswith** [comment]
자바스크립트의 스코프에 대해서 설명해주세요.

**Comment by jswith** [comment]
스코프체인이 무엇인가요?

**Comment by rladpwl0512** [comment]
식별자를 중복해서 사용할 수 있는 이유는 무엇인가요?

**Comment by rladpwl0512** [comment]
전역과 지역에 같은 변수명의 x가 선언되어있을 때, 지역에서 변수 x를 호출했을 때 전역과 지역의 변수 x 중 어떤 변수를 참조하는가? 
이유는?

**Comment by rladpwl0512** [comment]
동적 스코프, 렉시컬 스코프란? 
Javascript는 무엇을 채택하나요?
